data/sources/nz-cruise-association/nz-cruise-schedule.csv

```diff
@@ -0,0 +1,1124 @@
+Ship,Port,Arrival,Arrival time,Departure,Departure time,Guests (Lower Berths),Crew
+Pacific Aria,Auckland,06/07/19,07:00:00,06/07/19,16:00:00,1260,597
+Pacific Aria,Auckland,16/07/19,07:00:00,16/07/19,16:00:00,1260,597
+Pacific Aria,Auckland,24/07/19,07:00:00,24/07/19,16:00:00,1260,597
+Pacific Aria,Auckland,02/08/19,07:00:00,02/08/19,16:00:00,1260,597
+Pacific Aria,Auckland,05/08/19,07:00:00,05/08/19,16:00:00,1260,597
+Sea Princess,Auckland,25/08/19,10:00:00,25/08/19,21:00:00,2016,839
+Sea Princess,Bay Of Islands,26/08/19,08:00:00,26/08/19,17:00:00,2016,839
+Sea Princess,Auckland,29/09/19,10:00:00,29/09/19,20:00:00,2016,839
+Sea Princess,Bay Of Islands,30/09/19,07:00:00,30/09/19,16:00:00,2016,839
+Maasdam,Fiordland (cancelled),30/09/19,08:00:00,30/09/19,17:00:00,1258,580
+Maasdam,Dunedin,01/10/19,10:00:00,01/10/19,18:00:00,1258,580
+Maasdam,Lyttelton,02/10/19,08:00:00,02/10/19,17:00:00,1258,580
+Majestic Princess,Fiordland,03/10/19,07:00:00,03/10/19,16:00:00,3560,1357
+Maasdam,Kaikoura (cancelled),03/10/19,08:00:00,03/10/19,17:00:00,1258,580
```

Comment:
Can we have the status as a separate column?

Reply:
@whistlebee Yep, again with your spot of the headers not in camelCase, we can put together a richer version of the NZ Cruise Schedule. I assume you mean to separate the `(cancelled)` to its own column? Also the dates need to be brought into line with the `dd/mm/yyyy` format. I'll do this now 😄

Reply:
@whistlebee requested changes now committed, a new cleaned up cruise ship schedule can now be found in `data/meta`